setup.py

```diff
@@ -1,39 +1,28 @@
 #!/usr/bin/env python
 """Setup file for python-matlab-bridge"""
 
-import platform
 import os
 import sys
 import shutil
+import glob
+from distutils.core import setup
+
 
 # BEFORE importing distutils, remove MANIFEST. distutils doesn't properly
 # update it when the contents of directories change.
 if os.path.exists('MANIFEST'):
     os.remove('MANIFEST')
 
-from distutils.core import setup
 
-# Find the messenger binary file and copy it to /matlab folder.
+# Find the messenger binary file(s) and copy it to /matlab folder.
+sys.path.insert(0, 'messenger')
```

Comment:
Would we? `setup.py` is at the same level as `messenger`, and imports in the same directory normally take precedence (unless there's something special about packaging I'm not familiar with).